tests/example.py

```diff
@@ -1,31 +1,27 @@
 from platform import system
 from tkinter import Text, Tk
-from tkinter.font import Font
 from tkinter.ttk import Style
 
-if system() == "Darwin":
+if system() in ("Darwin", "Windows"):
```

Comment:
```suggestion
if system() == "Darwin":
```
Probably a mistake, lol. Windows doesn't have Command

Reply:
Yeah, I was fixing the scroll inversion and accidentally changed this too.